dask_planner/src/sql/optimizer.rs

```diff
@@ -45,8 +38,9 @@ impl DaskSqlOptimizer {
     /// optimizers as well as any custom `OptimizerRule` trait impls that might be desired.
     pub fn new() -> Self {
         debug!("Creating new instance of DaskSqlOptimizer");
+
         let rules: Vec<Arc<dyn OptimizerRule + Sync + Send>> = vec![
-            Arc::new(TypeCoercion::new()),
+            // Arc::new(TypeCoercion::new()),
```

Comment:
Was there a specific issue with this rule enabled?

Reply:
TypeCorercion is no longer a OptimizerRule. Instead it was moved to be an Analyzer. Will just remove the commented out line entirely